helm/defectdojo/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "2.1.0-dev"
+appVersion: "2.2.0-dev"
 description: A Helm chart for Kubernetes to install DefectDojo
 name: defectdojo
 version: 1.6.10
```

Comment:
I think we should merge that PR even though the version of the helm chart needs to be updated. We can create another PR with the updated chart version.

Reply:
I fear that would break the unit tests. Could be incorrect though. @valentijnscholten @madchap what are your thoughts?

Reply:
I usually bump it, so 1.6.11